patches/server/0978-Don-t-enforce-icanhasbukkit-default-if-alias-block-e.patch

```diff
@@ -0,0 +1,24 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: TheMeinerLP <p.glanz@madfix.me>
+Date: Tue, 13 Jun 2023 16:10:59 +0200
+Subject: [PATCH] Don't enforce icanhasbukkit default if alias block exists
+
+
+diff --git a/src/main/java/org/bukkit/craftbukkit/CraftServer.java b/src/main/java/org/bukkit/craftbukkit/CraftServer.java
+index ffa27c9c02dc4d12411fc089de3af8e8e12ba06e..30cad700d5278a2c5e79eecfd526c7ceb13aa5bd 100644
+--- a/src/main/java/org/bukkit/craftbukkit/CraftServer.java
++++ b/src/main/java/org/bukkit/craftbukkit/CraftServer.java
+@@ -351,8 +351,11 @@ public final class CraftServer implements Server {
+             legacyAlias = null;
+         }
+         this.commandsConfiguration = YamlConfiguration.loadConfiguration(this.getCommandsConfigFile());
+-        this.commandsConfiguration.options().copyDefaults(true);
```

Comment:
This line cannot be removed, defaults still have to be copied, we just update the defaults to not include the icanhasbukkit block if aliases exists.